{"uid":"36b65597b7f3efdc","name":"TC04-Add Label in the new created table","fullName":"testCases.test_Table.TestHome#test_Add_NewLabel","historyId":"b2f0718d5ffd929e1f770e57a0e624a7","time":{"start":1760502357163,"stop":1760502359710,"duration":2547},"status":"failed","statusMessage":"AssertionError: Typing failed in 'Name' after 2 attempts: Locator.fill: Error: Element is not an <input>, <textarea>, <select> or [contenteditable] and does not have a role allowing [aria-readonly]\nCall log:\n  - waiting for locator(\"//div[@class=\\\"fieldNameInput__input\\\"]\").first\n    - locator resolved to <div class=\"fieldNameInput__input\">…</div>\n    - fill(\"Name\")\n  - attempting fill action\n    - waiting for element to be visible, enabled and editable","statusTrace":"self = <pageObjects.Table_page.TablePage object at 0x7ff4ce9c5bd0>\nlocator = <Locator frame=<Frame name= url='https://team.quickbase.com/nav/app/bvhcuf7ef/table/bvhvb6a94/action/ListFields'> selector='//div[@class=\"fieldNameInput__input\"]'>\ntext = 'Name', field_name = 'Name', timeout = 5000, retries = 1\n\n    def type(self, locator, text: str, field_name: str = None, timeout=5000, retries=1):\n        \"\"\"Type text (auto-waits, supports string or Locator, retries once on failure).\"\"\"\n        step_name = field_name if field_name else locator\n        display_text = self.mask_sensitive(step_name, text)\n        with allure.step(f\"Type value '{display_text}' into {step_name} input field\"):\n            attempt = 0\n            while attempt <= retries:\n                try:\n                    element = self._wait_for_element(locator, timeout)\n>                   element.fill(text)\n\npageObjects/BasePage.py:56: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/sync_api/_generated.py:15889: in fill\n    self._sync(\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_locator.py:212: in fill\n    return await self._frame.fill(self._selector, strict=True, **params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_frame.py:590: in fill\n    await self._fill(**locals_to_params(locals()))\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_frame.py:602: in _fill\n    await self._channel.send(\"fill\", self._timeout, locals_to_params(locals()))\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_connection.py:69: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x7ff4cfd43dd0>\ncb = <function Channel.send.<locals>.<lambda> at 0x7ff4ce9b9bc0>\nis_internal = False, title = None\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False, title: str = None\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(\n            task, \"__pw_stack__\", None\n        ) or inspect.stack(0)\n    \n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal, title)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.Error: Locator.fill: Error: Element is not an <input>, <textarea>, <select> or [contenteditable] and does not have a role allowing [aria-readonly]\nE           Call log:\nE             - waiting for locator(\"//div[@class=\\\"fieldNameInput__input\\\"]\").first\nE               - locator resolved to <div class=\"fieldNameInput__input\">…</div>\nE               - fill(\"Name\")\nE             - attempting fill action\nE               - waiting for element to be visible, enabled and editable\n\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_connection.py:558: Error\n\nDuring handling of the above exception, another exception occurred:\n\nself = <testCases.test_Table.TestHome object at 0x7ff4cfef19d0>\n\n    @allure.title(\"TC04-Add Label in the new created table\")\n    def test_Add_NewLabel(self):\n>       TablePage(self.page).Add_NewLabel()\n\ntestCases/test_Table.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npageObjects/Table_page.py:68: in Add_NewLabel\n    self.type(self.addLabel, \"Name\", \"Name\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pageObjects.Table_page.TablePage object at 0x7ff4ce9c5bd0>\nlocator = <Locator frame=<Frame name= url='https://team.quickbase.com/nav/app/bvhcuf7ef/table/bvhvb6a94/action/ListFields'> selector='//div[@class=\"fieldNameInput__input\"]'>\ntext = 'Name', field_name = 'Name', timeout = 5000, retries = 1\n\n    def type(self, locator, text: str, field_name: str = None, timeout=5000, retries=1):\n        \"\"\"Type text (auto-waits, supports string or Locator, retries once on failure).\"\"\"\n        step_name = field_name if field_name else locator\n        display_text = self.mask_sensitive(step_name, text)\n        with allure.step(f\"Type value '{display_text}' into {step_name} input field\"):\n            attempt = 0\n            while attempt <= retries:\n                try:\n                    element = self._wait_for_element(locator, timeout)\n                    element.fill(text)\n                    return\n                except Exception as e:\n                    if attempt < retries:\n                        time.sleep(1)\n                        attempt += 1\n                    else:\n>                       raise AssertionError(f\"Typing failed in '{field_name}' after {retries + 1} attempts: {e}\")\nE                       AssertionError: Typing failed in 'Name' after 2 attempts: Locator.fill: Error: Element is not an <input>, <textarea>, <select> or [contenteditable] and does not have a role allowing [aria-readonly]\nE                       Call log:\nE                         - waiting for locator(\"//div[@class=\\\"fieldNameInput__input\\\"]\").first\nE                           - locator resolved to <div class=\"fieldNameInput__input\">…</div>\nE                           - fill(\"Name\")\nE                         - attempting fill action\nE                           - waiting for element to be visible, enabled and editable\n\npageObjects/BasePage.py:63: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1760502310679,"stop":1760502310679,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"setup_and_teardown","time":{"start":1760502310680,"stop":1760502312822,"duration":2142},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"_verify_url","time":{"start":1760502310680,"stop":1760502310680,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"pytestconfig","time":{"start":1760502310680,"stop":1760502310680,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"delete_output_dir","time":{"start":1760502310680,"stop":1760502310680,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"log_on_failure","time":{"start":1760502357163,"stop":1760502357163,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: Typing failed in 'Name' after 2 attempts: Locator.fill: Error: Element is not an <input>, <textarea>, <select> or [contenteditable] and does not have a role allowing [aria-readonly]\nCall log:\n  - waiting for locator(\"//div[@class=\\\"fieldNameInput__input\\\"]\").first\n    - locator resolved to <div class=\"fieldNameInput__input\">…</div>\n    - fill(\"Name\")\n  - attempting fill action\n    - waiting for element to be visible, enabled and editable","statusTrace":"self = <pageObjects.Table_page.TablePage object at 0x7ff4ce9c5bd0>\nlocator = <Locator frame=<Frame name= url='https://team.quickbase.com/nav/app/bvhcuf7ef/table/bvhvb6a94/action/ListFields'> selector='//div[@class=\"fieldNameInput__input\"]'>\ntext = 'Name', field_name = 'Name', timeout = 5000, retries = 1\n\n    def type(self, locator, text: str, field_name: str = None, timeout=5000, retries=1):\n        \"\"\"Type text (auto-waits, supports string or Locator, retries once on failure).\"\"\"\n        step_name = field_name if field_name else locator\n        display_text = self.mask_sensitive(step_name, text)\n        with allure.step(f\"Type value '{display_text}' into {step_name} input field\"):\n            attempt = 0\n            while attempt <= retries:\n                try:\n                    element = self._wait_for_element(locator, timeout)\n>                   element.fill(text)\n\npageObjects/BasePage.py:56: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/sync_api/_generated.py:15889: in fill\n    self._sync(\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_locator.py:212: in fill\n    return await self._frame.fill(self._selector, strict=True, **params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_frame.py:590: in fill\n    await self._fill(**locals_to_params(locals()))\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_frame.py:602: in _fill\n    await self._channel.send(\"fill\", self._timeout, locals_to_params(locals()))\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_connection.py:69: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x7ff4cfd43dd0>\ncb = <function Channel.send.<locals>.<lambda> at 0x7ff4ce9b9bc0>\nis_internal = False, title = None\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False, title: str = None\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(\n            task, \"__pw_stack__\", None\n        ) or inspect.stack(0)\n    \n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal, title)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.Error: Locator.fill: Error: Element is not an <input>, <textarea>, <select> or [contenteditable] and does not have a role allowing [aria-readonly]\nE           Call log:\nE             - waiting for locator(\"//div[@class=\\\"fieldNameInput__input\\\"]\").first\nE               - locator resolved to <div class=\"fieldNameInput__input\">…</div>\nE               - fill(\"Name\")\nE             - attempting fill action\nE               - waiting for element to be visible, enabled and editable\n\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_connection.py:558: Error\n\nDuring handling of the above exception, another exception occurred:\n\nself = <testCases.test_Table.TestHome object at 0x7ff4cfef19d0>\n\n    @allure.title(\"TC04-Add Label in the new created table\")\n    def test_Add_NewLabel(self):\n>       TablePage(self.page).Add_NewLabel()\n\ntestCases/test_Table.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npageObjects/Table_page.py:68: in Add_NewLabel\n    self.type(self.addLabel, \"Name\", \"Name\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pageObjects.Table_page.TablePage object at 0x7ff4ce9c5bd0>\nlocator = <Locator frame=<Frame name= url='https://team.quickbase.com/nav/app/bvhcuf7ef/table/bvhvb6a94/action/ListFields'> selector='//div[@class=\"fieldNameInput__input\"]'>\ntext = 'Name', field_name = 'Name', timeout = 5000, retries = 1\n\n    def type(self, locator, text: str, field_name: str = None, timeout=5000, retries=1):\n        \"\"\"Type text (auto-waits, supports string or Locator, retries once on failure).\"\"\"\n        step_name = field_name if field_name else locator\n        display_text = self.mask_sensitive(step_name, text)\n        with allure.step(f\"Type value '{display_text}' into {step_name} input field\"):\n            attempt = 0\n            while attempt <= retries:\n                try:\n                    element = self._wait_for_element(locator, timeout)\n                    element.fill(text)\n                    return\n                except Exception as e:\n                    if attempt < retries:\n                        time.sleep(1)\n                        attempt += 1\n                    else:\n>                       raise AssertionError(f\"Typing failed in '{field_name}' after {retries + 1} attempts: {e}\")\nE                       AssertionError: Typing failed in 'Name' after 2 attempts: Locator.fill: Error: Element is not an <input>, <textarea>, <select> or [contenteditable] and does not have a role allowing [aria-readonly]\nE                       Call log:\nE                         - waiting for locator(\"//div[@class=\\\"fieldNameInput__input\\\"]\").first\nE                           - locator resolved to <div class=\"fieldNameInput__input\">…</div>\nE                           - fill(\"Name\")\nE                         - attempting fill action\nE                           - waiting for element to be visible, enabled and editable\n\npageObjects/BasePage.py:63: AssertionError","steps":[{"name":"Add Label in the new created table","time":{"start":1760502357163,"stop":1760502359709,"duration":2546},"status":"failed","statusMessage":"AssertionError: Typing failed in 'Name' after 2 attempts: Locator.fill: Error: Element is not an <input>, <textarea>, <select> or [contenteditable] and does not have a role allowing [aria-readonly]\nCall log:\n  - waiting for locator(\"//div[@class=\\\"fieldNameInput__input\\\"]\").first\n    - locator resolved to <div class=\"fieldNameInput__input\">…</div>\n    - fill(\"Name\")\n  - attempting fill action\n    - waiting for element to be visible, enabled and editable\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/quickBase/quickBase/pageObjects/Table_page.py\", line 68, in Add_NewLabel\n    self.type(self.addLabel, \"Name\", \"Name\")\n  File \"/home/runner/work/quickBase/quickBase/pageObjects/BasePage.py\", line 63, in type\n    raise AssertionError(f\"Typing failed in '{field_name}' after {retries + 1} attempts: {e}\")\n","steps":[{"name":"Click on element - Add Label","time":{"start":1760502357163,"stop":1760502357688,"duration":525},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Type value 'Name' into Name input field","time":{"start":1760502357688,"stop":1760502359709,"duration":2021},"status":"failed","statusMessage":"AssertionError: Typing failed in 'Name' after 2 attempts: Locator.fill: Error: Element is not an <input>, <textarea>, <select> or [contenteditable] and does not have a role allowing [aria-readonly]\nCall log:\n  - waiting for locator(\"//div[@class=\\\"fieldNameInput__input\\\"]\").first\n    - locator resolved to <div class=\"fieldNameInput__input\">…</div>\n    - fill(\"Name\")\n  - attempting fill action\n    - waiting for element to be visible, enabled and editable\n\n","statusTrace":"  File \"/home/runner/work/quickBase/quickBase/pageObjects/BasePage.py\", line 63, in type\n    raise AssertionError(f\"Typing failed in '{field_name}' after {retries + 1} attempts: {e}\")\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":2,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":3,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"log_on_failure::0","time":{"start":1760502359850,"stop":1760502359910,"duration":60},"status":"passed","steps":[],"attachments":[{"uid":"6c55ad78554b04e","name":"failed_test","source":"6c55ad78554b04e.png","type":"image/png","size":55781}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":false,"hasContent":true},{"name":"setup_and_teardown::0","time":{"start":1760502496813,"stop":1760502496893,"duration":80},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"parentSuite","value":"testCases"},{"name":"suite","value":"test_Table"},{"name":"subSuite","value":"TestHome"},{"name":"host","value":"runnervmwhb2z"},{"name":"thread","value":"3583-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"testCases.test_Table"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"1f91bed5ab178d5d","status":"failed","statusDetails":"AssertionError: Typing failed in 'Name' after 2 attempts: Locator.fill: Error: Element is not an <input>, <textarea>, <select> or [contenteditable] and does not have a role allowing [aria-readonly]\nCall log:\n  - waiting for locator(\"//div[@class=\\\"fieldNameInput__input\\\"]\").first\n    - locator resolved to <div class=\"fieldNameInput__input\">…</div>\n    - fill(\"Name\")\n  - attempting fill action\n    - waiting for element to be visible, enabled and editable","time":{"start":1760502351291,"stop":1760502354966,"duration":3675}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"36b65597b7f3efdc.json","parameterValues":[]}